### PR TITLE
TCM-530 | On Demand Player updates

### PR DESCRIPTION
**Pull request by devender197**
### Summary

- [x] Deleted the unused asset
- [x] VideoActivity.kt name refactor to VideoPlayer.kt

### Issue

[TCM-530](https://hyfnla.atlassian.net/browse/TCM-530)

### Video/Screenshot
Screenshot_1624058886

### CODE REVIEW

**Review by onodude — CHANGES_REQUESTED** [review]
so video activity should be `VideoPlayerActivity`

**Comment by devender197** [comment]
> so video activity should be `VideoPlayerActivity`

[Done](https://github.com/hyfn/trx-consumer-android/pull/123/commits/a535f51294b1007c8ec28b514c7b34c1d3d21d05)

**Review by onodude — DISMISSED** [review]
thanks!

File: app/src/main/java/com/trx/consumer/screens/video/VideoPlayerActivity.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 
-class VideoActivity : BrightcovePlayer() {
+class VideoPlayerActivity : BrightcovePlayer() {
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/screens/video/VideoPlayerActivity.kt:35]: Let's rename the layout file name as well to match

**Review by VivekV95 — CHANGES_REQUESTED** [review]
One more naming change

File: app/src/main/java/com/trx/consumer/screens/video/VideoPlayerActivity.kt (hunk 1 above)
Comment (devender197) [reply app/src/main/java/com/trx/consumer/screens/video/VideoPlayerActivity.kt:35]: [Done](https://github.com/hyfn/trx-consumer-android/pull/123/commits/67949b02819083d0bf0e1c71675017ce10cdcc33#diff-b951aad0aee5eec1b805bbf5fa9d7eef32758105002df16c115e6a44ce8298bbR48)

**Review by onodude — APPROVED** [review]
thanks!

**Review by VivekV95 — APPROVED** [review]

**Pull request merged**